### ISSUE TITLE
Build parameters

### ISSUE THREAD

**Issue by szepeviktor** [allinurl/goaccess#1068]
```
$ goaccess --version
GoAccess - 1.2.
For more details visit: http://goaccess.io
Copyright (C) 2009-2016 by Gerardo Orellana
```
Many C programs display the `./configure` parameters in version info.
Please consider adding [info](https://github.com/allinurl/goaccess/blob/master/configure.ac#L286-L297) (openssl, debug, utf8, geoip, storage, compression) to `--version` output.

**Comment by allinurl** [comment]
For some reason I thought somebody asked about this before, but I couldn't find it. I can certainly do that. Thanks!

**Comment by allinurl** [comment]
Implemented #1101.

**Comment by szepeviktor** [comment]
Thanks
